components/ShareRoom/ShareRoomSidebarList.tsx

```
@@ -0,0 +1,51 @@
+export default function ShareRoomSidebarList() {
+	const { currentUser } = useContext(authContext);
+	const userId = currentUser?.uid;
+	const [shareRooms, setShareRooms] = useState<ShareRooms[]>([]);
+	const dispatch = useDispatch<AppDispatch>();
+
+	if (!userId) {
+		return <div>ログインが必要です</div>;
+	}
+
+	useEffect(() => {
+		const fetchShareRoomData = async () => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: dispatch

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)